### PR TITLE
#862 Shipment related links

### LINKED ISSUE #862: Add related document links to invoice pages
`InvoiceNode` now has some fields for related documents: `linkedShipment` and `requisition` which can be linked to from the `related documents` section of the side bar

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #862 

- Adds links for shipments to the a requisition

### CODE REVIEW

File: packages/invoices/src/InboundShipment/DetailView/SidePanel/AdditionalInfoSection.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { memo } from 'react';
```
Comment (joshxg) [inline packages/invoices/src/InboundShipment/DetailView/SidePanel/AdditionalInfoSection.tsx:1]: I also moved the inbound sections in the side panel out. Just copy pasted except for the related docs section

File: packages/invoices/src/InboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const d = useFormatDate();
+  const { requisition } = useInboundFields('requisition');
+
+  let tooltip = '';
+  if (requisition) {
+    const { user, createdDatetime } = requisition;
+    tooltip = t('messages.internal-order-created-on', {
+      date: d(new Date(createdDatetime)),
+    });
+    if (user?.username && user.username !== 'unknown') {
```
Comment (mark-prins) [inline packages/invoices/src/InboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx:28]: that would be useful as a utility function - though the full thing would require the translation hook too, which is a pain.

**Review by mark-prins — APPROVED** [review]
👍  awesome!

File: packages/invoices/src/InboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx (hunk 2 above)
Comment (joshxg) [reply packages/invoices/src/InboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx:28]: Hmm you're right it's a lot of duplication. What about a `UserTooltip`? Should that use the info icon or another? I'l make an issue

**Pull request merged**